src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Process/Process.ts

```diff
@@ -29,7 +32,10 @@ export var detailDirective = (
         restrict: "E",
         templateUrl: adhConfig.pkg_path + pkgLocation + "/Detail.html",
         scope: {
-            path: "@"
+            path: "@",
+            isBuergerhaushalt: "=?",
+            isKiezkasse: "=?",
+            contentType: "=?"
```

Comment:
`contentType` should use `"@"`.

Also: It seems to be redundant to have `contentType` **and** `isBuergerhaushalt`/`isKiezkasse`.